CHANGELOG

```diff
@@ -2,6 +2,10 @@
 
 Release notes for the fsalum-newrelic module.
 
+#### 5.1.0 - 2024-06-04
+
+* Add newrelic_php_extension_enabled option
```

Comment:
Would be nice if this would also indicate what the option is for, f.e. 
```
* Add possibility to install and configure newrelic php agent without loading the extension. This allows loading the extension from within php on request using php's dl() function.
```